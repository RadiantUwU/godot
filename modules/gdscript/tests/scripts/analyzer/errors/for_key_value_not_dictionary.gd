func test():
    var a := [1, 2, 3]
    for k, v in a:
        prints(k, v)
