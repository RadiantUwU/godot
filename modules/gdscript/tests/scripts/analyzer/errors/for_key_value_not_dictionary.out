GDTEST_ANALYZER_ERROR
Unable to iterate "Array" as a key-value pair.
